docs/pages/get-started/react-redux.mdx

```diff
@@ -228,12 +246,14 @@ Get people’s cursor positions with
 `liveblocks.others.map(user => user.presence?.cursor)`. It’s worth noting that a
 user presence can be `undefined`.
 
-```jsx
+```tsx
 import { useSelector } from "react-redux";
+import {LiveBlocksState} from "./store";
 
 function OthersCursors() {
-  const others = useSelector((state) => state.liveblocks.others);
-
+  const others = useSelector<LiveBlocksState, LiveBlocksState["others"]>(
+    (state) => state.liveblocks.others
+  );
```

Comment:
Correct! This type annotation is unfortunately still necessary with the current version of `@liveblocks/redux`. The goal is to eventually not need this and instead let TypeScript be able to derive this type automatically, just like with our React and Zustand packages. But for now, this is indeed the way to type it 👍